styles/auth.scss

```diff
@@ -0,0 +1,110 @@
+.form-wrap {
+  width: 32rem;
+  margin: 3rem auto;
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
```

Comment:
If you used this box-shadow elsewhere you can make it a helper class

Reply:
Ok will do that when I refactor the whole code